### ISSUE TITLE
expected scalar type Half but found Byte

### ISSUE THREAD

**Issue by cube3power** [josStorer/RWKV-Runner#293]
{"detail":"failed to load: The following operation failed in the TorchScript interpreter.\nTraceback of TorchScript (most recent call last):\n File \"D:\\rwkv\\backend-python\\rwkv_pip\\model.py\", line 1928, in v6_0_before\n sx = torch.cat((sx.unsqueeze(0), xx[:-1, :])) - xx\n xxx = xx + sx * x_maa\n xxx = torch.tanh(xxx @ tm_w1).view(T, 5, -1).transpose(0, 1)\n ~~~~~~~~~~~ <--- HERE\n xxx = torch.bmm(xxx, tm_w2).view(5, T, -1)\n mw, mk, mv, mr, mg = xxx.unbind(dim=0)\nRuntimeError: expected scalar type Half but found Byte\n"}

**Comment by josStorer** [comment]
try [v1.7.1](https://github.com/josStorer/RWKV-Runner/releases/tag/v1.7.1)
